.github/workflows/pr.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.17
+          go-version: 1.18
```

Comment:
You may want to change the go version in ado pipeline as well.
https://github.com/Azure/aztfy/blob/main/azure-pipelines.yml#L21

Reply:
Indeed, thank you for the spot!